insights/specs/default.py

```diff
@@ -527,6 +527,7 @@ def semid(broker):
     multipath_conf = simple_file("/etc/multipath.conf")
     multipath_conf_initramfs = simple_command("/bin/lsinitrd -f /etc/multipath.conf")
     multipath__v4__ll = simple_command("/sbin/multipath -v4 -ll")
+    mysqladmin_status = simple_command("/bin/mysqladmin status")
```

Comment:
This spec requires that you include a simple_file spec in `insights/specs/insights_archive.py` so that insights-core can read the collected data from an insights archive.  Use the `mangle` command to determine the file path for the archive.